components/resume/CardDialog.tsx

```
@@ -72,49 +85,107 @@ const CardDialog = (props: Props): JSX.Element | null => {
-        >
-          <div className="w-4 h-4 flex">
-            <Image src={closeButton} alt="close" layout="intrinsic" />
-          </div>
-        </div>
-        <div className="flex items-center">
-          <div className="w-6 h-6 mr-2">
-            <Image src={creditCard} alt="card" layout="intrinsic" />
+        {/* Header Bar */}
+        <div className="flex items-center justify-between p-4 border-b border-gray-200 bg-gray-50">
+          <div className="flex items-center space-x-2">
+            <button className="flex items-center space-x-1 px-3 py-1 text-sm font-medium text-gray-700 rounded bg-gray-100 hover:bg-gray-200">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)